app/controllers/cron_controller.rb

```diff
@@ -29,7 +29,9 @@ def run
     when 'codes_expire'
       Code.cleanup_expired
     when 'sphinx_reindex'
-      system('rake', '--rakefile', Rails.root+'/Rakefile', 'ts:index', 'RAILS_ENV=production')
+      require 'rake'
```

Comment:
@elijh What do you think... does it make sense to run this rake task via the rails process?
:smiley: :
* no need to load the whole environment for a short lived task
:disappointed: : 
* cluttering the rails process with rake and rake tasks
* blocking one of the passenger processes